lib/spack/docs/getting_started.rst

```diff
@@ -87,6 +87,9 @@ for your shell:
    # For fish
    $ . spack/share/spack/setup-env.fish
 
+   # For nix
+   $ cd spack/share/spack; nix-shell
```

Comment:
Perhaps rather:
```suggestion
   $ nix-shell spack/share/spack/shell.nix
```
to preserve the current directory? It's also slightly different from the sh/csh/fish versions since it starts a new shell so it may deserve a separate block?

Reply:
:+1: